CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ Please carefully follow the whitespace and formatting conventions already presen
 ## Add Apache license header to all new classes
 ```ruby
 # Cloud Foundry Java Buildpack
-# Copyright 2013-2015 the original author or authors.
+# Copyright 2013-2016 the original author or authors.
```

Comment:
yeah, but its all upstream change, so I would rather just sync it up with them when the change is made ;)